src/Samples/AuthoringDemo/AuthoringDemo/AuthoringDemo.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+        <Platforms>x64</Platforms>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    </PropertyGroup>
+
+    <!-- CsWinRT properties -->
+    <PropertyGroup>
+        <CsWinRTComponent>true</CsWinRTComponent>
+        <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+        <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+        <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.1.1" />
+    </ItemGroup>
+
+    <!-- NuGet package generation properties -->
+    <PropertyGroup>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <Version>1.0.0</Version>
+        <PackageOutputPath>.\nuget</PackageOutputPath>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <_PackageFiles Include="AuthoringDemo.targets" PackagePath="build;buildTransitive" />
```

Comment:
We probably should use the content thing to include it here if possible to avoid using _PackageFiles.  Take a look at the change Joshua did.